cmd/lakectl/cmd/config/config.go

```diff
@@ -40,6 +40,7 @@ type configuration struct {
 			Region    string `mapstructure:"region"`
 			CatalogID string `mapstructure:"catalog_id"`
 		}
+		FixSparkPlaceholder bool `mapstructure:"fix_spark_placeholder"`
```

Comment:
Please document "Spark location placeholder", it might not be clear to readers.  Also I suspect it might not be unique to Spark, so maybe we can document what it is/does.

Reply:
Will do, I considered and eventually decided not to.
Thanks!